packages/react-core/src/components/MultipleFileUpload/MultipleFileUploadStatusItem.tsx

```diff
@@ -6,6 +6,11 @@ import { Button } from '../Button';
 import FileIcon from '@patternfly/react-icons/dist/esm/icons/file-icon';
 import TimesCircleIcon from '@patternfly/react-icons/dist/esm/icons/times-circle-icon';
 
+/** Renders a progress bar for each individual file that has been attempted to be uploaded,
+ * including the file name, file type, file size, and upload status. Each status item also
+ * has a "remove" button to remove individual items from the status list.
+ */
```

Comment:
Nit: I'm not sure quite how technical we really need to be here, but it might be good if we mention that this sub-component also can handle the reading of the file that it gets passed?

Reply:
Maybe something like, "Renders a progress bar for each individual file that has been attempted to be uploaded. This sub-component also handles reading the uploaded files to render the file name, file type, file size, and upload status." (may want to separate "upload status" from the rest? not sure if that would really tie into the file getting read or not)

Reply:
I like that! The only reservation I have about it is that I feel like (maybe) it implies that having it read the files is mandatory rather than something consumers can opt out of if they need custom file reading logic?

Again I might totally just be reading this wrong or reading into it too much 😅